package.json

```diff
@@ -8,10 +8,13 @@
     "dev": "nodemon --exec babel-node ./src/index ",
     "coverage": "nyc npm test",
     "start": "babel-node ./src",
-    "pretest": " NODE_ENV=test npm run migrate-undo && NODE_ENV=test npm run migrate && NODE_ENV=test npm run seed",
+    "pretest": " NODE_ENV=test npm run db-drop &&  NODE_ENV=test npm run db-create && NODE_ENV=test npm run migrate-undo && NODE_ENV=test npm run migrate && NODE_ENV=test npm run seed",
```

Comment:
Since you have dropped and created DB. you don't have to do migrate undo. This `npm run migrate-undo` can be removed in the pretest script.

Reply:
solved